app/src/main/java/com/android/streetworkapp/ui/progress/Progression.kt

```diff
@@ -370,3 +386,18 @@ sealed class DashboardStateProgression {
 
   data object Achievement : DashboardStateProgression()
 }
+
+fun exerciseNameToIcon(name: String): Int {
```

Comment:
The function is quite explicit, but a definition of the function and its parameters would be useful.